src/pages/myGeotab/guides/developingAddIns.tsx

```diff
@@ -0,0 +1,848 @@
+
+    <p>The contents of <code className="small-code-sample">console.log("...")</code> statements can be examined using the Developer Tools, the timeline of XML HTTP requests can be viewed, and breakpoints to step through the JavaScript source code can be created.</p>
+
+    <p>It is also recommended to read Google's extensive learning resources available on using <a href="https://developers.google.com/chrome-developer-tools/">Chrome Developer Tools</a> to get started debugging or learn about the advanced features they have available.</p>
+
+    <p><strong>Note:</strong> Add-Ins that were locally added to a database in the past (the recommended approach is to host on a server externally) might throw the error message <strong>"Add-In threw an error. Please contact your administrator."</strong> To further confirm, the following error will appear in the browser console (found by hitting CTRL+SHIFT+I): <strong>Error: Add-In files: &lt;add-in file name&gt; couldn't be loaded. Probably they were moved to another location or removed.</strong> This could be linked to a server maintenance/migration event. The workaround would be to re-upload the files for the Add-Ins back to the database, while the resolution would be to externally host the source code as per the requirements listed <a href="https://geotab.github.io/sdk/software/guides/developing-addins/#requirements">here</a>.</p>
+</div>;
+
+//
+const AddInIconValidator: ReactNode = <div className="paragraph">
+    <p>The Add-In Icon Validator tests uploaded SVG files against <a href="https://www.geotab.com/blog/mygeotab-add-in-icons-specs/">Geotab's requirements</a> and displays them in the reformatted colors and style that will be displayed in MyGeotab. <a href="../developing-addins/addin-icon-validator.html">Launch the validator</a>.</p>
+</div>;
+
+
+export default function DevelopingAddIns() {
+    return (
+        <div className="grayBackground pageContent">
+            <h1 className="title">Developing Add-Ins</h1>
+            <div className="paragraph">
+                <p>Add-Ins are used to extend the functionality provided by MyGeotab and Geotab Drive. An Add-In is JavaScript, HTML and CSS loaded into the MyGeotab or Geotab Drive portal and resides directly inside the user interface. This allows third-parties to create a seamless user experience and provide solutions that would otherwise require the user to visit a different website altogether. <a href="https://github.com/Geotab/sdk-addin-samples">Click here</a> to find the sample Add-Ins.</p>
+                <div className="side-note">
+                    <p>The add-in generator is a great developer tool that allows integrators to create scaffolded add-in projects. You can learn more about the generator and all of its features by going into the <a href="https://github.com/Geotab/generator-addin">generator-addin repository</a>.</p>
```

Comment:
for the sections with the left-hand-side blue bar, as discussed in standup today we'll put those in the blue informational boxes as soon as that component is ready from the myG team. Can you add TODOs for now where all of those go so they're easy to fix later?

Reply:
Created a component for informational box